### ISSUE TITLE
A11y_AzureToolsStorageExplorer_ConnectToAzureStorage_ScreenReader: Screen reader not announces expanded/collapsed state for 'Azure environments' combo box.

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#2411]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #NVDA; #JAWS; #Win32; #CELAAssessID_CosmosDB-FY20-Q2; 

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.11.1
Windows10

**ReproSteps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. "Connect to Azure Storage" popup opens.
4.  Navigate to "Azure Environments" combo box.
5. Using Enter collapse and expand the 'Azure environments:' combo box.

**Actual:**
When NVDA and JAWS focus are on 'Azure environments:' combo box within the 'Connect to Azure Storage' dialog, the expanded/collapsed state are not properly announced.

**Same issue is observed in below Scenarios:**
1. Attach to local emulator.
2. Use Storage Account name and key, 
3. Add a resource via Azure Active Directory.
4. Disk (Upload VHD).

**Expected:**
The expanded/collapsed state of the 'Azure environments:' combo box should always be announced to the user to inform them that they have successfully expanded or collapsed a combo box.

**UserImpact:**
Screen reader users may not be able to know the exact State of the Combo box whether it is in expanded state or collapsed state.

**Comment by MRayermannMSFT** [comment]
None of the major browsers are currently doing this behavior (on Windows). I've opened an issue against Chromium: https://bugs.chromium.org/p/chromium/issues/detail?id=1035513&can=2&q=OS%3D%22Mac%22%20component%3AUI%3EAccessibility%3ECompatibility&sort=-status%20-pri

**Comment by Alexandria-R** [comment]
Approved for exclusion as low impact.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
